traffic_ops/testing/api/v4/servercapabilities_test.go

```diff
@@ -89,6 +90,41 @@ func ValidationTestServerCapabilities(t *testing.T) {
 	}
 }
 
+func UpdateTestServerCapabilities(t *testing.T) {
+	var header http.Header
+
+	// Get server capability name and edit it to a new name
+	resp, _, err := TOSession.GetServerCapabilitiesWithHdr(header)
+	if err != nil {
```

Comment:
Could you add a check for `len(resp) != 1` here, so that we can avoid chances of a panic on line 101/ 103?

Reply:
In GetServerCapabilityWithHdr(), the check is already present and hence I didn't add the check.

Reply:
Done